server/templates/home.html

```diff
@@ -4,92 +4,79 @@
 {% block title %}Home{% endblock %}
 
 {% block content %}
-<div class="container">
-  <div class="bs-component">
-    <div class="row">
-      {% if rulesDocs != None %}
-      <div class="col-lg-3 col-md-3 col-sm-4">
+<div class="row justify-content-between">
 
-        {{ list.holoListOfLinks(rulesDocs) }}
+  {% for navList in navLists %}
+  <div>
 
-      </div>
-      {% endif %}
+    {{ list.holoListOfLinks(navList)}}
 
-      {% if character_docs != None %}
-      <div class="col-lg-3 col-md-3 col-sm-4">
+  </div>
+  {% endfor %}
 
-        {{ list.holoListOfLinks(character_docs) }}
 
-      </div>
-      {% endif %}
+  {# MEMBERS SECTION #}
+  {% if 'username' in session %}
 
-      {% if itemsDocs != None %}
-      <div class="col-lg-3 col-md-3 col-sm-4">
+  {# Not sure how many of these are needed. For now, they are at most only for members. #}
```

Comment:
if you're not sure, ask on slack!
It would be nice if the NPC generator and Files page were available without logging in. The files page has our old character sheet, and hopefully will have our new one soon. Not everyone who plays CX has a login yet. You're right, it would be nice to have a register button.

Reply:
There's a lot of uglifying of the front end before it gets prettier, so sometimes I just put things aside to worry about later, and I'm trying to organize according to the logic in my head and forget that players are actually using this site.

Players without accounts can still access /files and /npcgen, there just isn't explicit navigation.